src/app/index.tsx

```diff
@@ -1,19 +1,22 @@
 import React from 'react';
-import { View, StatusBar } from 'react-native';
+import { View, StatusBar, FlatList, ListRenderItem } from 'react-native';
 import Book from '@app/components/Book';
+import { BookProps } from '@interfaces/books';
+import { BOOK_LIST } from '@constants/books';
 
 import styles from './styles';
 
 const App = () => {
+  const keyExtractor = (item: BookProps, _index: number) => item.title;
```

Comment:
The `index` is not necessary, also use id for `keyExtractor` => `const keyExtractor = ({ id }: BookProps) => String(id);`